pyiron/atomistics/structure/atoms.py

```diff
@@ -306,6 +308,24 @@ def elements(self):
         """
         return np.array([self.species[el] for el in self.indices])
 
+    def get_high_symmetry_points(self):
```

Comment:
Just a small description in docstrings will be good

Reply:
done